src/pages/optimized-for-change/index.zh-hans.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 为迎接改变优化
+date: '2018-12-12'
+spoiler: 什么造就了优秀的API?
+---
+
+什么造就了优秀的API？
+
+*优秀的* API 设计是便于记忆以及清晰不含糊的。它能促使代码变得可读性强，执行正确，以及性能优异, 也能帮助开发者进入[成功之坑](https://blog.codinghorror.com/falling-into-the-pit-of-success/)。
+
+我把这些设计层面叫做 “第一级”，因为这些是一个库开发者需要关注的优先事项。你可能需要在它们中的某些上做妥协和权衡，但是至少这些都是你的考虑点。
```

Comment:
```suggestion
我把这些设计层面叫做 「第一级」，因为这些是一个库开发者需要关注的优先事项。你可能需要在它们中的某些上做妥协和权衡，但是至少这些都是你的考虑点。
```
This is just my suggestion. Using square brackets will increase readability. Also see reference [here](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-CN.md#%E7%AE%80%E4%BD%93%E4%B8%AD%E6%96%87%E4%BD%BF%E7%94%A8%E7%9B%B4%E8%A7%92%E5%BC%95%E5%8F%B7).